### PR TITLE
Add pcap, rpm, systemd packages to main image.

### PR DESCRIPTION
**Pull request by mikemadden42**

### CODE REVIEW

**Comment by mikemadden42** [comment]
Related issue:
https://github.com/elastic/golang-crossbuild/issues/23

File: go1.10/main/Dockerfile.tmpl [hunk 1]
```diff
@@ -12,6 +12,12 @@ RUN \
         gcc-multilib \
         libc6-dev \
         libc6-dev-i386 \
+        libpcap-dev \
```
Comment (andrewkroh) [inline go1.10/main/Dockerfile.tmpl:15]: Packetbeat builds libpcap from source due to some issues we had to resolve:

- https://github.com/elastic/beats/blob/abae05768c4c445bddffd6fff520a62ccca2e126/packetbeat/magefile.go#L260-L262
- https://github.com/elastic/beats/pull/4217

I think you will have to do the same thing in the image if you fully want to make the image fully self-contained. Did you already test building Packetbeat with this image using the OS provided libpcap?

Addressing libpcap could be left until the end. It's not really a blocker for the issue you are trying to resolve. Having it in the image will speed up the builds, but we don't utilize any `apt` commands from Packetbeat's build to get libpcap built.

Comment (mikemadden42) [reply go1.10/main/Dockerfile.tmpl]: I removed the `libpcap` libraries for now.  We can reassess it at the end when all the other packages are addressed.

**Comment by mikemadden42** [comment]
I'm running into an issue building the image after adding the `librpm` and `libsystemd-dev` packages.

```
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:
The following packages have unmet dependencies:
 librpm-dev : Depends: librpm3 (= 4.11.3-1.1) but it is not going to be installed
              Depends: librpmio3 (= 4.11.3-1.1) but it is not going to be installed
              Depends: librpmbuild3 (= 4.11.3-1.1) but it is not going to be installed
              Depends: librpmsign1 (= 4.11.3-1.1) but it is not going to be installed
              Conflicts: librpm-dev:i386 but 4.11.3-1.1 is to be installed
 librpm-dev:i386 : Depends: libdb-dev:i386 but it is not going to be installed
                   Conflicts: librpm-dev but 4.11.3-1.1 is to be installed
E: Unable to correct problems, you have held broken packages.
```

**Comment by mikemadden42** [comment]
If I remove `librpm-dev:i386`, the build succeeds.

**Comment by andrewkroh** [comment]
You're running into a mutliarch conflicts. I think the only way to solve this is to have separate images for `amd64` and `386`. You're going to hit those issues with the arm build image too.

**Comment by mikemadden42** [comment]
I'm going to close this PR. I've started with a better approach in this PR:
https://github.com/elastic/golang-crossbuild/pull/53

**Pull request closed without merging**